docs/configuration/configuration.md

```diff
@@ -451,6 +451,26 @@ metric_relabel_configs:
 # native histogram. If this is exceeded, the entire scrape will be treated as
 # failed. 0 means no limit.
 [ native_histogram_bucket_limit: <int> | default = 0 ]
+
+# Lower limit for the growth factor of one bucket to the next in each native histogram. 
+# The resolution of a histogram with a lower growth factor will be reduced until it is within the limit.
+# Here is the conversion table from growth factor to resulting schema/scale.
```

Comment:
Wordsmithing: "To set an upper limit for the schema (equivalent to "scale" in OTel's exponential histograms), use the following factor limits:"